Hush/Assets/Scripts/Game/SpellSelectorManager.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using UnityEngine;
+using Weapon;
+using Weapon.Enums;
+
+namespace Game
+{
+    public class SpellSelectorManager : MonoBehaviour
+    {
+        public static SpellSelectorManager Instance;
+        [SerializeField] private GameObject player;
```

Comment:
If you want to be consistent with the rest of the Managers get the player via Tag or FindObjectOfType

It juste makes the managers easier to use, never need to set references in the inspector